bash/fzf-bash-completion.sh

```diff
@@ -391,16 +393,16 @@ _fzf_bash_completion_complete() {
                 (
                     unset COMP_WORDS COMP_CWORD
                     export COMP_LINE="$COMP_LINE" COMP_POINT="$COMP_POINT" COMP_KEY="$COMP_KEY" COMP_TYPE="$COMP_TYPE"
-                    compl_command="$(eval "echo $compl_command")"
+                    compl_command="$(eval "printf '%s\n' $compl_command")"
```

Comment:
similar to the other one, this is going to print each on a separate line instead of space separated

Reply:
:+1: fixed